prdoc/prdoc/pr_6621.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Update Conviction Voting Pallet to Support Block Number Provider
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR makes the conviction voting pallet uses the relay chain as a block provider for a parachain on a regular schedule.
```

Comment:
It doesn't force the conviction voting pallet to use the relay chain block provider,
```suggestion
      This PR makes the block number provider used in the conviction voting pallet configurable. Before this PR, conviction voting pallet always used the system block number, with this PR some runtime can opt to use the relay chain block number instead.
```